src/server/models/db/migrations/20230531144729_add_related_links/migration.sql

```diff
@@ -0,0 +1,55 @@
+update "List"
+set "jsonData" = jsonb_set("jsonData", '{relatedLinks}',
+    case "type"
+        when 'lawyers'
+        then coalesce("jsonData"->'relatedLinks', '[]') || '[
+            {
+                "url": "gov.uk/guidance/arrested-or-detained-abroad",
```

Comment:
could you prepend with https:// please? 🙏

Reply:
Yep, that makes sense